src/app/root-content/dashboard/widgets/documents/documents-widget.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="content widget-header">Documents
+	<div class="right floated">
+		<i class="refresh small icon widget-button" (click)="refreshWidget()"></i>
+		<i class="remove small icon widget-button" (click)="closeWidget()"></i>
+	</div>
+</div>
+<div class="content widget-body">
+	<div class="widget-text">{{documentsTotalCount}}</div>
+	<img class="widget-icon" src="/assets/images/documentIcon.png" height="60" *ngIf="!errorMessage"/>
```

Comment:
Use CSS's `background` property to set the image as background (which seems to be the intended purpose of the widget's images).